camelAssemblies/src/main/resources/sourceConfigOverrides/jira_add_comment_sink/v4_4_3/jira_add_comment_sink.json

```diff
@@ -0,0 +1,14 @@
+{
+  "camelRuntime": {
+  },
+  "general": {
+    "repoList": [
+      "https://repo.maven.apache.org/maven2/",
+      "https://packages.atlassian.com/mvn/maven-external/"
+    ],
```

Comment:
These libraries, when depdendency detected from the jira-specific repo, seem to get their versions modified (_mediated_ in ivy terms) away.  This causes them to fail to load.  Just fixing things locally here.